core-libs/setup/ssr/logger/loggers/server-logger.factory.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { inject } from '@angular/core';
+import { DefaultServerLogger } from './default-server-logger';
+import { serverLoggerToken } from './server-logger';
+
+/**
+ * Factory provides logger that's used for pre-rendering purposes.
```

Comment:
```suggestion
 * Returns a `ServerLogger` that can be used by a `LoggerService`.
 * 
 * If any specific `ServerLogger` implementation has been already provided
 * on the platform injector level (e.g. in ExpressJS app), we use it.
 * Otherwise we fallback to the `PrerenderigServerLogger`.
```